topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -0,0 +1,444 @@
+>
+>
+>    {% snippet faqs/galaxy/histories_import.md %}
+>
+> 2. **Rename** {% icon galaxy-pencil %} the the history to your name of choice.
+>
+{: .hands_on}
+
+> <hands-on-title>Option 2: Data upload - Add to history</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the AnnData object from [Zenodo]({{ page.zenodo_link }})
+>
+>    ```
+>    {{ page.zenodo_link }}/files/Final_cell_annotated_object.h5ad
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. **Rename** {% icon galaxy-pencil %} the .h5ad object as `Final cell annotated object`
+>
+>    {% snippet faqs/galaxy/datasets_rename.md name="Final cell annotated object" %}
+>
+> 4. Check that the datatype is `h5ad`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="h5ad" %}
+>
+{: .hands_on}
+
+# Important tips for easier analysis
+
+{% snippet faqs/galaxy/tutorial_mode.md %}
+
+> <comment-title></comment-title>
+> - The Galaxy tool search panel sometimes doesn't find the tools we need from the thousands available.
+> - You'll have a much easier time selecting tools from the panel (if you aren't using tutorial mode!) if you are on the [humancellatlas.usegalaxy.eu](https://humancellatlas.usegalaxy.eu)
+{: .comment}
+
+## Filtering for T-cells
+
+One problem with our current dataset is that it's not just T-cells: we found in the previous tutorial that it also contains macrophages. This is a problem, because trajectory analysis will generally try to find relationships between all the cells in the sample. We need to remove those cell types to analyse the trajectory.
+
+> <hands-on-title> Removing macrophages </hands-on-title>
+>
+> 1. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.7.5+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `output` (Input dataset)
+>    - *"Function to manipulate the object"*: `Filter observations or variables`
+>        - *"What to filter?"*: `Observations (obs)`
+>        - *"Type of filtering?"*: `By key (column) values`
+>            - *"Key to filter"*: `cell_type`
+>            - *"Type of value to filter"*: `Text`
+>                - *"Filter"*: `not equal to`
+>                - *"Value"*: `Macrophages`
+>                - 
+> 3. **Rename** {% icon galaxy-pencil %} output h5ad `T-cell_object.h5ad`
+>
+{: .hands_on}
+
+You should now have `8569` cells, as opposed to the `8605` you started with. You've only removed a few cells (the contaminants!), but it makes a big difference in the next steps.
+
+## Draw force-directed graph
+
+First, we will calculate a [force-directed graph](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.draw_graph.html), as an alternate to tSNE, which will likely work better for trajectory analysis.
```

Comment:
maybe append "(FDG)" here too, for those who don't put together the acronym in their heads